mlserver/parallel/dispatcher.py

```diff
@@ -90,7 +97,15 @@ def _get_worker(self) -> Worker:
         By default, this is just a round-robin through all the workers.
         """
         worker_pid = next(self._workers_round_robin)
-        return self._workers[worker_pid]
+        return self._workers[worker_pid], worker_pid
```

Comment:
We may need to update to update the return signature of this method to `Tuple[Worker, int]` (linter will complain otherwise)